src/unitStats/weaponLib.ts

```
@@ -317,15 +317,31 @@ const getSingleWeaponDPS = (
-  const dps = (finalRpm / 60) * (finalHitChance * finalDamage + aoeDamageCombines);
+  const dps = (finalRpm / 60) * (finalHitChance * finalDamage);
 
   return dps * weapon_member.num;
 };
 
-export const getWeaponRpm = (weapon_bag: WeaponStatsType, distance = 0, isMoving = false) => {
+export const getWeaponRpm = (weapon_bag: WeaponStatsType, distance = 0, isMoving = false, attacking_unit?: CustomizableUnit,) => {
   // average aim time
-
-  const aimTime = getInterpolationByDistance(
+  let aimMod=1,windMod=1,cooldownMod=1,reloadMod=1,burstLength=1,burstShots=1;
```

ESLint

[error] 'burstLength' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'burstLength' is never reassigned. Use 'const' instead.

(prefer-const)